parts/chapters/subsections/6.3/HMMLT.fodt

```diff
@@ -4154,7 +4154,7 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
      </table:table-row>
     </table:table>
     <text:h text:style-name="P18404" text:outline-level="4"><text:bookmark-start text:name="__RefHeading___Toc14509_370116838816216"/>Description<text:bookmark-end text:name="__RefHeading___Toc14509_370116838816216"/></text:h>
-    <text:p text:style-name="_40_TextBody">The HMMLT series of keywords defines the history match gradient cumulative permeability multipliers, for when the History Match Gradient option has been activated by the HMDIMS keyword in the RUNSPEC section. The keyword consists of the first five characters of “HMMLT” followed by a two or three character string shown in <text:sequence-ref text:reference-format="category-and-value" text:ref-name="refTable119">Table 6.45</text:sequence-ref>, that determines the permeability direction, for example, HMMLTPX.</text:p>
+    <text:p text:style-name="_40_TextBody">The <text:a xlink:href="#__RefHeading___Toc224680_4219267791">HMMLT</text:a> series of keywords defines the history match gradient cumulative permeability multipliers, for when the History Match Gradient option has been activated by the <text:a xlink:href="#__RefHeading___Toc207753_4219267791">HMDIMS</text:a> keyword in the <text:a xlink:href="#__RefHeading___Toc55591_1778172979">RUNSPEC</text:a> section. The keyword consists of the first five characters of “HMMLT” followed by a two or three character string shown in <text:sequence-ref text:reference-format="category-and-value" text:ref-name="refTable119">Table 6.45</text:sequence-ref>, that determines the permeability direction, for example, HMMLTPX.</text:p>
```

Comment:
Yes. We discussed this earlier, see: https://github.com/OPM/opm-reference-manual/pull/411#discussion_r1833035716 and we decided not to link keywords within quotes. In #427 the script is updated so that keywords in single quotes are not linked either.